pkg/vulnsrc/ubuntu/ubuntu.go

```diff
@@ -213,3 +217,17 @@ func SeverityFromPriority(priority string) types.Severity {
 		return types.SeverityUnknown
 	}
 }
+
+// StatusFromUbuntuStatus normalises Ubuntu status into common Trivy Types
+func StatusFromUbuntuStatus(status string) types.Status {
+	switch status {
+	case "ignored":
+		return types.StatusWillNotFix
```

Comment:
I took another read of the document.
https://askubuntu.com/questions/1509705/cve-questions-about-status/1509706#1509706

>"ignored" support has ended for one reason or another/ end-of-life.

It looks like Ubuntu doesn't determine if the package is affected. So, if I understand correctly, we should not show this type of vulnerability.

Reply:
We consider this an intentional decision that is `vendor will not fix`, this status is important in our downstream workflow and was included for this reason.

Reply:
"Will not fix" means "this package is affected by this flaw on this platform, but there is currently no intention to fix it." It's a Red Hat definition, but we follow it.
https://access.redhat.com/blogs/product-security/posts/2066793

Reply:
@knqyf263 Are you happy to proceed with this?
What do you need from me to get this merged?

Reply:
@skahn007gl - it seems that treating ignored as wontfix is the issue here, as ignored/EOL findings are typically not ingested to reduce Trivy DB size. I think we can just skip ingesting ignored, we still have the "notaffected" status to determine if the outcome of analysis is that Ubuntu decided not to fix it. For EOL stuff, we can work that out another way on our end, and treat findings for EOL releases as wontfix if they don't have another status.

Reply:
Trivy is designed to minimize false positives as much as possible. EOL means unknown and is possibly a false positive, which is why we currently do not detect it.

Reply:
While i see the reasoning behind this and do not disagree, I am happy to follow guidance and remove the status for ubuntu.

@knqyf263 let me know if you feel strongly on removing `ignored` for Ubuntu and i will make the changes.

I would like to highlight the `ignored` is ingested for Debian in `pkg/vulnsrc/debian.go:682`
```
func newStatus(s string) types.Status {
	switch strings.ToLower(s) {
	// "end-of-life" is considered as vulnerable
	// e.g. https://security-tracker.debian.org/tracker/CVE-2022-1488
	case "no-dsa", "unfixed":
		return types.StatusAffected
	case "ignored":
		return types.StatusWillNotFix
	case "postponed":
		return types.StatusFixDeferred
	case "end-of-life":
		return types.StatusEndOfLife
	}
	return types.StatusUnknown
} 
```

Reply:
>I would like to highlight the ignored is ingested for Debian in pkg/vulnsrc/debian.go:682

We believe "ignored" in Debian differs from "ignored" in Ubuntu. They use the same term by chance. Please let us know if you find a document in which Debian also uses "ignored" for EOL.

Reply:
Hey @knqyf263, @jhebden-gl 
I have removed `ignored` status as we can make an assumption on `wontfix` by presence of data (Affected), Deferred status, No fix version.